platform/utm_mgrs_utils.cpp

```diff
@@ -0,0 +1,567 @@
+#include "platform/utm_mgrs_utils.hpp"
+
+#include "base/string_utils.hpp"
+
+#include <string>
+#include <iostream>
+#include <cmath>
+
+namespace utm_mgrs_utils
+{
+using namespace std;
+
+typedef struct UTMPoint_Value
+{
+  double easting;
+  double northing;
+  int zone_number;
+  char zone_letter;
+
+} UTMPoint;
+
+UTMPoint latlon_to_utm(double lat, double lon);
+int latlon_to_zone_number(double lat, double lon);
+char latitude_to_zone_letter(double lat);
+int zone_number_to_central_longitude(int zone_number);
```

Comment:
These function names do not follow our code convention.
```suggestion
int ZoneNumberToCentralLongitude(int zone_number);
```